test/vars/UtilSpec.groovy

```diff
@@ -434,6 +434,6 @@ class UtilSpec extends JenkinsPipelineSpecification {
         when:
         def checkForFormatError = groovyScript.getNextSnapshot('a.12.0')
         then:
-        checkForFormatError == 'Version is not in the format X.Y.Z'
+        checkForFormatError == 'Version is not in the required format contains alphabets'
```

Comment:
here you will need to check that the `error` is called with correct message

Reply:
In this , I am checking that thing only whether error is called with the correct message by using "==" . If I am wrong do correct me

Reply:
Result from the PR check is
```
[ERROR] Failures: 
[ERROR]   UtilSpec.[util.groovy] calcNextSnapshot:438 Condition not satisfied:

checkForFormatError == 'Version is not in the required format contains alphabets'
|                   |
null                false
```

`error` is a Jenkins command and you should check the output:


```
    def "[util.groovy] calcNextSnapshot"() {
        when:
        def checkForFormatError = groovyScript.getNextSnapshot('a.12.0')
        then:
        1 * getPipelineMock("error")({it == "Version is not in the required format contains alphabets"})
    }  
```